hooks/command

```diff
@@ -1,16 +1,20 @@
 #!/bin/bash
 set -euo pipefail
 
-if [[ "$(arch)" == "aarch64" ]]; then
+arch="$(arch)"
+arm=("arm armhf aarch64 aarch64_be armv6l armv7l armv8l arm64e") # arm64
+amd=("x86 x86pc i386 i686 i686-64 x64 x86_64 x86_64h athlon")    # amd64
+if [[ "${arm[*]}" =~ ${arch} ]]; then
   arch="arm64"
-else
+elif [[ "${amd[*]}" =~ ${arch} ]]; then
   arch="amd64"
 fi
 
 plugin=${BUILDKITE_PLUGINS:-""}
 version=$(echo $plugin | sed -e 's/.*monorepo-diff-buildkite-plugin//' -e 's/\".*//')
+kernel=$(uname -s | awk '{print tolower($0)}')
```

Comment:
Can we have an explicit allow list or map of kernels. We pre-build the binaries are part of the release process so we  only support a limited set of kernels strings. Currently this is limited to `darwin` and `linux`.

See https://github.com/chronotc/monorepo-diff-buildkite-plugin/releases/tag/v2.2.0

Depending on the resulting `kernel` value we may need to map multiple values to `linux`. Alternatively we explicitly check for the `darwin` kernel and default to `linux`.